### PR TITLE
-Some word smithing, typos and additional late feedback type changes.

### PR DESCRIPTION
**Pull request by arnom-ms**

### CODE REVIEW

File: contributor/outline.md [hunk 1]
```diff
@@ -83,18 +84,19 @@ Outline for this section inspired by the [Trusted Committer introduction video].
...
     It's OK to ask for help.
-    You aren't expected to be as knowledgable and proficient as you might be on the full-time projects that your team owns.
-    - Match the existing coding style, indentation, etc, even if it's not what you would normally do and even if it is not specified in the `CONTRIBUTING.md`.
-    A guest contribution for a feature is not the time to introduce a new way of structuring or formatting project code.
+    You aren't expected to be as knowledgeable and proficient as you might be on the full-time projects that your team owns.
+    - Match the existing coding style, indentation, etc., even if it's not what you would normally do and even if it is not specified in the `CONTRIBUTING.md`.
+    A guest contribution for a feature or bug fix is not the time to introduce a new way of structuring or formatting project code.
   - Submitting the pull request
-    - Run the project build and automated validation before opening a pull request.
+    - Add tests for your code contributions to help making the code maintainable over time.
```
Comment (rrrutledge) [inline contributor/outline.md]: Does this point belong here as blanket advice that we give to _all_ contributors 

\- or -

Does this point belong in a project's `CONTRIBUTING.md` (which this outline already references)

?

File: contributor/outline.md [hunk 2]
```diff
@@ -12,19 +12,21 @@ Outline for this section inspired by the [Trusted Committer introduction video].
 ## Becoming an InnerSource Contributor
 
   - A contributing mindset starts with a sharing mindset.
-    - When you have technical needs that are orthogonal to the domain of your project,
-    look around to see if there is a shared solution that you can use.
+    - When you start adding new functionality to your project, ask yourself if this is new functionality (at least partially) 
+    reflects a general issue that needs a solution, or if this new functionality has technical needs that are orthogonal to the domain of your project.
```
Comment (lenucksi) [inline contributor/outline.md]: isn't there an 'and' missing?
```suggestion
    and reflects a general issue that needs a solution, or if this new functionality has technical needs that are orthogonal to the domain of your project. 
```

File: contributor/outline.md (hunk 1 above)
Comment (lenucksi) [reply contributor/outline.md]: I think that might actually be 'blanket advice' grade. 
Is there a realistic chance that any project could not want tested code?

Comment (rrrutledge) [reply contributor/outline.md]: I was thinking of textual contributions, but you're right it does say `code` contributions right there.  Sure, makes sense.

Comment (arnom-ms) [reply contributor/outline.md]: My motivation here is that this is a significant part of the contribution that may be overlooked specifically by less experienced people or people with little to no experience with open source.

File: contributor/outline.md (hunk 2 above)
Comment (arnom-ms) [reply contributor/outline.md]: Actually a comma is missing..

**Comment by lenucksi** [comment]
This looks good to me and like all open discussion points were resolved. Should we merge it in?

**Comment by rrrutledge** [comment]
> Should we merge it in?

Good with me, although if you both have a second I left [these suggestions](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/85) for this PR as well.

**Comment by lenucksi** [comment]
> > Should we merge it in?
> 
> Good with me, although if you both have a second I left [these suggestions](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/85) for this PR as well.

I think they're useful. After all, merging anything is a TC's call and this change encourage more people to consider contributing something. Maybe a TC could work with them to enhance the usefulness for others but at least a conversation was started. (With everything being a trade-off between available TC time for mentoring and desire for having contributions...but that's a different topic)

**Comment by lenucksi** [comment]
I'm going to merge this in now, such that handling #85 hopefully becomes easier to merge.

**Comment by rrrutledge** [comment]
Thanks, Johannes.  Great stuff!

**Pull request merged**